server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/CreateActionTest.java

```diff
@@ -117,43 +168,33 @@ public void return_created_profile_in_json() throws Exception {
       .setParam("name", "Yeehaw!")
       .execute();
 
-    JsonAssert.assertJson(response.getInput()).isSimilarTo(getClass().getResource("CreateActionTest/create-no-importer.json"));
+    JsonAssert.assertJson(response.getInput()).isSimilarTo(getClass().getResource("CreateActionTest/test_json.json"));
```

Comment:
I was sure you'll ask the question : the issue is that the WS is returning the profile key, which is generated randomly.